src/web/views/children-dob.njk

```diff
@@ -25,7 +26,8 @@
         legend: aboutYourChild,
         dateLegend: dateOfBirth,
         removeButton: removeChild
-      }
+      },
+      autofocus: i == children.inputCount - 1 and children.inputCount > 1
```

Comment:
Does this introduce it's own a11y issues? When a user with more than one child chooses to change details or use the back button, they will always be focused on the last input.

Reply:
Just thought of that and addressing that now.